test/arrayFill.test.js

```diff
@@ -0,0 +1,23 @@
+const { arrayFill } = require("../lib");
+
+describe("arrayFill", () => {
+  test("should return a new array", () => {
```

Comment:
Hey @amoradi !! Thanks so much for the contrib. We really appreciate it. So I believe the `.toBe` methods here should actually be `.toEqual`:

```javascript
  test("some stuff", () => {
    var foo = [1, 6, 7, "cake"];
    var bar = [1, 6, 7, "cake"];

    expect(bar).not.toBe(foo);
  });
```

The above test will pass, even though it should not.

```javascript
  test("should return a new array", () => {
    var foo = [1, 6, 7, "cake"];
    var bar = [1, 6, 7, "cake"];

    expect(bar).not.toEqual(foo);
  });
```

However ^^ This test will fail with the return of:

```
    Expected value to not equal:
      [1, 6, 7, "cake"]
```

I love the method tho! Nice work